lib/mintel/dashboards/components/panels/cost-analysis.libsonnet

```diff
@@ -464,8 +467,8 @@ local seriesOverrides = import 'components/series_overrides.libsonnet';
         },
       ],
       query=|||
-        (sum by (namespace) (namespace_name:kube_pod_container_resource_requests_cpu_cores:sum) * ($costcpu - ($costcpu / 100 * $costDiscount)) )
```

Comment:
I'm not sure this is right

These pre-recorded metrics do seem to exist, but this has a typo:

It's meant to be

```
namespace:kube_pod_container_resource_requests_cpu_cores:sum
```

Assuming the pre-recorded rule is correct we should prefer them (faster)

Reply:
https://github.com/mintel/kube-monitoring-mixins/blob/master/rules/prometheus-rules.yaml#L139

Reply:
Maybe a little bit of confusion here too, since that pre-recorded rule is already summed by namespace.

Reply:
Oh yeah that makes more sense. I'll give that a try!

Reply:
Have fixed these to use the pre-recorded values!